set1/Q3.cpp

```diff
@@ -0,0 +1,46 @@
+//https://practice.geeksforgeeks.org/problems/next-larger-element/0
+#include <iostream>
+#include<stack>
+
+using namespace std;
+long long int arr[10000001];
+
+void printNGE(int n){
+	stack<long long int> s;
+	for(int i=0; i<n; i++){
+
+		while(!s.empty() && arr[i] > arr[s.top()]){			
+			cout << arr[i] << " ";
```

Comment:
**Note:**
Some interviewers dont like a while inside for/while, be prepared to modify the code on the spot.